db/seeds.rb

```diff
@@ -0,0 +1,15 @@
+User.create!(name: "Example User",
+             email: "example@railstutorial.org",
+             admin: true,
+             password: "foobar",
+             password_confirmation: "foobar")
+# Generate a bunch of additional users.
+99.times do |n|
```

Comment:
fake ít thôi em, nhiều rake chạy chậm và dư thừa
```suggestion
30.times do |n|
```